core-graphics/src/event.rs

```diff
@@ -97,6 +98,25 @@ impl KeyCode {
     pub const RIGHT_ARROW: CGKeyCode = 0x7C;
     pub const DOWN_ARROW: CGKeyCode = 0x7D;
     pub const UP_ARROW: CGKeyCode = 0x7E;
+    pub const NUMPAD_0: CGKeyCode = 0x52;
```

Comment:
Could we use the convention in original [Ref](https://github.com/phracker/MacOSX-SDKs/blob/master/MacOSX10.13.sdk/System/Library/Frameworks/Carbon.framework/Versions/A/Frameworks/HIToolbox.framework/Versions/A/Headers/Events.h)?

Reply:
I agree, we should use the same name as the original reference. In addition, would it be possible to add *all* the missing key constants here and put them in the order of the original header file with comments indicating what section they came from?

Reply:
I added more constants, but I put them into new structs. Otherwise the comments would have been wrong. I also used the same names as in the original header file.